packages/@coorpacademy-app-player/src/middlewares/test/error-logger.js

```diff
@@ -0,0 +1,54 @@
+import test from 'ava';
+import {createStore} from 'redux';
+import createMiddleware from '../index';
+import macro from '../../actions/test/helpers/macro';
+
+test('should call ErrorLogger', async t => {
+  t.plan(1);
+
+  const options = {
+    services: {
+      Logger: {
+        error: () => {
+          t.pass();
+        }
+      }
+    }
+  };
+
+  const store = createStore(
+    state => {
+      return state;
+    },
+    'state',
+    createMiddleware(options)
+  );
+
+  await store.dispatch({type: 1, error: true, payload: {}});
+});
+
+const exception = new Error('Something wrong');
+
+test(
+  'should dispatch error using Logger',
```

Comment:
`should dispatch an error action when Logger.error() throws`

Rajoute un test pour vérifier que lorsque Logger.error() marche correctement, il n'y a pas d'action de dispatché.